src/main/java/org/jenkinsci/plugins/vsphere/builders/Deploy.java

```diff
@@ -116,6 +170,16 @@ private boolean deployFromTemplate(final AbstractBuild<?, ?> build, final Launch
 
         vsphere.deployVm(expandedClone, expandedTemplate, linkedClone, resourcePoolName, expandedCluster, expandedDatastore, powerOn, jLogger);
 		VSphereLogger.vsLogger(jLogger, "\""+expandedClone+"\" successfully deployed!");
+		IP = vsphere.getIp(vsphere.getVmByName(expandedClone), 20);
```

Comment:
True that, timeouts should be configurable, but this PR scope is allowing pipeline jobs invoking the vSphere plugin.

I have bumped that hardcoded timeout to 60 seconds (and that would try 12 times) and added a not null check for IP